### PR TITLE
Move `≡-setoid` to `Function.Indexed.Relation.Binary.Equality`

### PR DESCRIPTION
**Pull request by Saransh-cpp**
Related to #759 #1996

I could not get rid of the `_⟨$⟩_` imports from `Function.Equality`. Removing the occurrence of `_⟨$⟩_` in the files was not as straightforward as I initially thought 🙂

### CODE REVIEW

File: src/Function/Equality.agda [hunk 1]
```diff
@@ -11,6 +11,7 @@
 -- `Bijection`. The alternative definitions found in this file will
 -- eventually be deprecated.
 
+-- check if modules are actually using Function.Equality and port them
```
Comment (MatthewDaggitt) [inline src/Function/Equality.agda:14]: Could we get rid of these temporary notes before merging?

**Review by MatthewDaggitt — COMMENTED** [review]
Otherwise looks good! Thanks.

File: src/Function/Equality.agda (hunk 1 above)
Comment (Saransh-cpp) [reply src/Function/Equality.agda:14]: Oh oops, those are my notes, pushed them by mistake. Thanks!

**Comment by Saransh-cpp** [comment]
Thanks for the review. @MatthewDaggitt! Should this be documented in the CHANGELOG somewhere?

**Comment by MatthewDaggitt** [comment]
> Should this be documented in the CHANGELOG somewhere?

No that's okay.

**Pull request merged**